cpp/src/arrow/array.h

```diff
@@ -488,27 +489,33 @@ class ARROW_EXPORT BinaryArray : public FlatArray {
               const std::shared_ptr<Buffer>& null_bitmap = NULLPTR,
               int64_t null_count = 0, int64_t offset = 0);
 
-  // Return the pointer to the given elements bytes
-  // TODO(emkornfield) introduce a StringPiece or something similar to capture zero-copy
-  // pointer + offset
+  /// Return the pointer to the given elements bytes
+  // XXX should GetValue(int64_t i) return a string_view?
```

Comment:
Might be nice to just use string_view, assuming there aren't any microperformance implications

Reply:
We can do that in a later PR if desired.

Reply:
:+1: